search-app/src/cmr/search/api/concepts_lookup.clj

```diff
@@ -119,6 +120,9 @@
       {params :params headers :headers ctx :request-context}
       ;; XXX REMOVE this check and the stubs once the service and
       ;;     the associations work is complete
-      (if (headers "cmr-prototype-umm")
-        (stubs/handle-prototype-request path-w-extension params headers)
+      (if (= "true" (string/lower-case (headers "cmr-prototype-umm")))
```

Comment:
Wouldn't this throw NullPointerException for regular requests where `cmr-prototype-umm` header is not provided?

Reply:
Yup -- good catch.

Reply:
Fixed.